### ISSUE TITLE
Crafting menu doesn't open

### ISSUE THREAD

**Issue by Roxanya** [ZaneYork/SMAPI-Android-Installer#552]
The menu to access farmer info and inventory, crafting things, map, characters, achievements, settings, etc just doesn't open when i tap it. Help me pls, how can i submit my error log?

**Comment by Roxanya** [comment]
[Uploading SMAPI-latest.txt…]()

**Comment by Zencee** [comment]
> The menu to access farmer info and inventory, crafting things, map, characters, achievements, settings, etc just doesn't open when i tap it. Help me pls, how can i submit my error log?

resend your log but this time wait for it to fully load before sending

**Comment by Roxanya** [comment]
[SMAPI-latest.txt](https://github.com/ZaneYork/SMAPI-Android-Installer/files/12438068/SMAPI-latest.txt)

**Comment by Zencee** [comment]
there are so many mods that you've installed that have been skipped due to lack of necessary mods for them to work but lets just skip that since they're not the ones whats causing you the problems that you stated..

problems:
1. DGA isnt supported for mobile yet so it will have some errors along the way which causes spacecore to give more errors in your log.
2. Json assets couldn't load a lot of your files namely jewel crafting since theres an invalid png file inside, forest amber which is also invalid, Artisan valley machines it contains duplicate files so smapi would not load any of it, also idk what is this but it seems like a korean mod you should check on it on your files since json assets couldn't load them.
3. Flower valley- a lot of this mods assets couldn't be loaded mostly due to because of missing pngs.
4. [cp]Eugene mod failed to load due to having a duplicate mod which does the same thing as this mod, delete the eugene mod and only use the translation one.
5. Producer framework mod couldn't load the amber resin mod as well as files th at starts with wf? PFM sweet tooth has no config file.
6. And some mods that uses Farm type manager is giving you some critical errors which i don't have a clue which is it so im afraid you will have to solve it yourself by removing mods.

there are a lot of errors and im guessing you added them in bulk instead of testing them 3-5 at a time lmao well goodluck on solving these issues

**Comment by Haerinah** [comment]
> > there are so many mods that you've installed that have been skipped due to lack of necessary mods for them to work but lets just skip that since they're not the ones whats causing you the problems that you stated..
> > problems:
> > 
> > 1. DGA isnt supported for mobile yet so it will have some errors along the way which causes spacecore to give more errors in your log.
> > 2. Json assets couldn't load a lot of your files namely jewel crafting since theres an invalid png file inside, forest amber which is also invalid, Artisan valley machines it contains duplicate files so smapi would not load any of it, also idk what is this but it seems like a korean mod you should check on it on your files since json assets couldn't load them.
> > 3. Flower valley- a lot of this mods assets couldn't be loaded mostly due to because of missing pngs.
> > 4. [cp]Eugene mod failed to load due to having a duplicate mod which does the same thing as this mod, delete the eugene mod and only use the translation one.
> > 5. Producer framework mod couldn't load the amber resin mod as well as files th at starts with wf? PFM sweet tooth has no config file.
> > 6. And some mods that uses Farm type manager is giving you some critical errors which i don't have a clue which is it so im afraid you will have to solve it yourself by removing mods.
> > 
> > there are a lot of errors and im guessing you added them in bulk instead of testing them 3-5 at a time lmao well goodluck on solving these issues
> 
> Tysm, i will try to fix the things you pointed out and cut some mods that are less important to my game experience. So DGA wouldn't work at all, even if it loads? How sad, i looked forward to try those decorations :( Since my pc modded game works so slow, i prefer playing on mobile for time issues.

yes DGA loads but it will give you a lot of errors probably due to incompatibility with the mobile smapi. Its the same thing with dynamic reflections, they'll give you too many errors and once your phone couldn't handle those pilled up errors your game would most likely crash once you sleep and save.

**Comment by Roxanya** [comment]
hmmm...So does it can still partially work to some certain degree?
Nevermind, i somehow solved the mess by creating a new save. It wasn't a thing of DGA because after i erased all them the button still failed on a save, but it was on a new.
Thank you all for your tips!
Happy farming! Gotta enjoy those 3k of ja of Atelier Wildflour mods, at least.
